### PR TITLE
Update gemspec, bump version

### PR DESCRIPTION
**Pull request by mble**
This commit updates the `gemspec` to point towards to the repository,
and makes it clear that this is a Heroku gem.

### CODE REVIEW

File: logplex.gemspec [hunk 1]
```diff
@@ -6,11 +6,11 @@ require 'logplex/version'
 Gem::Specification.new do |gem|
   gem.name          = "logplex"
   gem.version       = Logplex::VERSION
-  gem.authors       = ["Harold Giménez"]
+  gem.authors       = ["Harold Giménez", "Heroku"]
   gem.email         = ["harold.gimenez@gmail.com"]
```
Comment (treznick) [inline logplex.gemspec:10]: should we update this?

**Review by treznick — APPROVED** [review]

File: logplex.gemspec (hunk 1 above)
Comment (stevenharman) [reply logplex.gemspec:10]: Maybe a team mailing list or something?

**Review by stevenharman — APPROVED** [review]

File: logplex.gemspec (hunk 1 above)
Comment (mble) [reply logplex.gemspec:10]: Yes, the real question is to what!

Comment (mble) [reply logplex.gemspec:10]: Given this is a public gem, internal mailing lists are probably not the right call. I thought about `support@heroku.com`.

**Comment by mble** [comment]
Merging for now, we can deal with ownership later :)

**Pull request merged**